common/protob/messages-management.proto

```diff
@@ -58,6 +58,23 @@ message Features {
     optional bool unfinished_backup = 27;       // report unfinished backup (equals to Storage.unfinished_backup)
     optional bool no_backup = 28;               // report no backup (equals to Storage.no_backup)
     optional bool recovery_mode = 29;           // is recovery mode in progress
+    repeated Feature features = 30;             // list of supported features
+    enum Feature {
+        Feature_Bitcoin = 1;
+        Feature_Altcoins = 2;
```

Comment:
The term `Altcoin` is really ambiguous. I think it usually covers all other coins than Bitcoin. Maybe `Bitcoin_like`? Or let's just add a comment

Reply:
I'll add a comment

Reply:
Resolved in b729bc6f65bbff6d5c64f63d0014acb063c5f5ab